### DISCUSSION TITLE
blog/drug-blog-game-and-addiction

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#51] [Announcements]
# blog/drug-blog-game-and-addiction

Two thousand seven hundred and fifty words

https://pathos.page/blog/drug-blog-game-and-addiction

**Comment by savouer1** [comment]
生命的本质在于追求幸福，而幸福来源于理性。反过来说，如果你是理性的，那么你就是幸福的。这里就和你上面说的上瘾性行为相悖了，因为上瘾性行为在行为时本身也是幸福的，但它却不是理性的。我认为之所以这样，是还没有对幸福本身有清晰认知的结果，即上瘾性行为所产生的快感是一种伪幸福，它并没有满足你真正的欲望，而是遮掩了真实欲望。学习，成长的过程，就是不断探寻自己真实幸福的过程。在这个过程中，我认为上瘾性行为不是被意志所克服掉，而是在认知变化之后，你自然对它们失去了兴趣。

**Comment by lshcool** [comment]
文章质量高！
个人认为，有成瘾性就有成功性。
成瘾意味着专注、积极反馈。
前提是能够引导正面成瘾。

**Comment by petite-dd** [comment]
都写得非常好，但确乎太多了，刚才想仔细看看某一篇，怎么翻都没翻到尽头，才发现有2w多字😂
佩服博主是其一，让我认知到自己的专注度没这么高是其二
最后我只能留一个路过祝好～
